cursorless-talon/src/actions/get_text.py

```diff
@@ -7,11 +7,12 @@
 
 def cursorless_get_text_action(
     target: CursorlessTarget,
+    *,
     show_decorations: Optional[bool] = None,
     ensure_single_target: Optional[bool] = None,
```

Comment:
Make these boolean parameters keyword args to avoid [the boolean trap](https://github.com/pwoolvett/flake8_boolean_trap).